app/components/inputs/PathBrowseInput.js

```diff
@@ -0,0 +1,68 @@
+const electron = require("electron");
+const dialog = electron.remote.dialog;
+const mainWindow = electron.remote.getCurrentWindow();
+const ipc = electron.ipcRenderer;
+
+import PathInput from "./PathInput";
+import { PathButton } from "../buttons";
+
+var pathListener;
+var key;
```

Comment:
Don't use global variables, as adding two components at the same time will make the behaviour undefined (imagine the second component ovewriting the `key`/`listener` variable)